### DISCUSSION TITLE
Window statusline

### DISCUSSION THREAD

**Discussion by sarumont** [NvChad/NvChad#2233] [Q&A] [answered]
Is there a way to make the statusline window-centric and not global? I generally have several vertical splits open, and I can't tell at a glance what file is open in what window since the NVChad status line is global (unlike the default in neovim). The only thing I have been able to find related to this is [#9](https://github.com/NvChad/NvChad/issues/9), and it is so old as to not be relevant.

Thanks!

**Comment by siduck** [comment, marked as answer]
@sarumont there's a workaround. 

https://github.com/NvChad/ui/issues/26

**Reply by sunaku** [reply]
The workaround code snippet needs a little update to work with the current NvChad v2.0 branch, per https://github.com/NvChad/ui/issues/26#issuecomment-1681321168.
